README.md

```diff
@@ -50,7 +50,6 @@ You will get the line: `<svg xmlns="http://www.w3.org/2000/svg" ...`
 | ![IconChevronRight](dist/icons/IconChevronRight.svg) | `IconChevronRight` |
 | ![IconChevronUp](dist/icons/IconChevronUp.svg) | `IconChevronUp` |
 | ![IconClipboard](dist/icons/IconClipboard.svg) | `IconClipboard` |
-| ![IconCode](dist/icons/IconCode.svg) | `IconCode` |
```

Comment:
what is about code and inline-code icons?

Reply:
IconCode has been deleted.
There are IconBrackets and IconCurlyBrackets for that reason